config/config.exs

```diff
@@ -27,3 +27,8 @@ use Mix.Config
 # Configuration from the imported file will override the ones defined
 # here (which is why it is important to import them last).
 #
+
+config :glass_factory_api, GlassFactoryApi.ApiClient,
```

Comment:
WDYT you move this to the `test.exs` configuration file?
I think those values would only be useful in that scenario, right?
That way in case people forgot to set it in their `prod.secret.exs` or similar file.
That way you crash the application instead of sending requests to a strange place and leave people wondering what is going on.

Reply:
If I move to `test.exs` I will need a `dev.exs` to run locally too, right?

Reply:
Yes, you will.
Since those are not real information, I thought you already had some sort of `dev.secret.exs` file with real endpoint / key / email configuration in place. If you don't, that is probably a nice to have.